ENGDOCS-744 Compose Install Fixes

dockertopia opened the pull request:
- Changed headers case from title case to sentence case
- Fixed typos
- Changed compose version var

usha-mandya commented on compose/install/compose-plugin.md:
```diff
@@ -0,0 +1,194 @@
+>
+> These instructions assume you already have Docker Engine and Docker CLI installed and now want to install the Compose plugin. 
+>
+> Note as well this option requires you to manage upgrades manually. Whenever possible we recommend any of the other installation methods listed. For other Linux installation methods see [this summary](#installation-methods).
```
```suggestion
> Installing the Compose plugin manually also requires you to manage upgrades manually. Therefore, we recommend that you use the other installation methods documented in the [Installation methods](#installation-methods) section.
```

usha-mandya commented on compose/install/compose-plugin.md:
```diff
@@ -0,0 +1,194 @@
+
+> **Note**
+>
+>__Compose standalone__: If you need to use Compose without installing the Docker CLI, the instructions for the standalone scenario are similar. 
```
Should this be a new heading as we are providing instructions on how to install Compose as a standalone?

usha-mandya commented on compose/install/compose-plugin.md:
```diff
@@ -0,0 +1,194 @@
+on Microsoft Windows Server and want to install Docker Compose.
+
+
+1.  Run a PowerShell as an administrator. 
```
```suggestion
1.  Run PowerShell as an administrator. 
```

usha-mandya commented on compose/install/compose-plugin.md:
```diff
@@ -0,0 +1,194 @@
+
+
+1.  Run a PowerShell as an administrator. 
+When asked if you want to allow this app to make changes to your device, click **Yes** in order to proceed with the installation.
```
```suggestion
When asked if you want to allow this app to make changes to your device, click **Yes** to proceed with the installation.
```

usha-mandya commented on compose/install/index.md:
```diff
@@ -0,0 +1,38 @@
+[Install Docker Desktop for your platform](./compose-desktop.md). This is the fastest route and you get Docker Engine and Docker CLI with the Compose plugin. Docker Desktop is available for Mac, Windows and Linux.
+
+* __Install Compose plugin:__
+  + __(Mac, Win, Linux) Docker Desktop__: If you have Desktop installed then you already have the Compose plugin installed.
```
```suggestion
  + __Docker Desktop__: If you have Docker Desktop (Mac, Windows, Linux) installed, then you already have the Compose plugin installed.
```

usha-mandya commented on compose/install/index.md:
```diff
@@ -0,0 +1,38 @@
+* __Get latest Compose and its prerequisites right of the bat__:
+[Install Docker Desktop for your platform](./compose-desktop.md). This is the fastest route and you get Docker Engine and Docker CLI with the Compose plugin. Docker Desktop is available for Mac, Windows and Linux.
+
+* __Install Compose plugin:__
```
```suggestion
* __Install the Docker Compose plugin:__
```

usha-mandya commented on compose/install/uninstall.md:
```diff
@@ -0,0 +1,58 @@
+### Uninstalling Docker Desktop
+
+If you want to uninstall Compose and you have installed Docker Desktop, follow the corresponding link bellow to get instructions on how to remove Docker Desktop. 
+> Please note that, unless you have other Docker instances installed in that specific environment, you would be removing Docker altogether by uninstalling the Desktop.
```
Can we format this as a Note?

usha-mandya commented on compose/install/uninstall.md:
```diff
@@ -0,0 +1,58 @@
+If you want to uninstall Compose and you have installed Docker Desktop, follow the corresponding link bellow to get instructions on how to remove Docker Desktop. 
+> Please note that, unless you have other Docker instances installed in that specific environment, you would be removing Docker altogether by uninstalling the Desktop.
+
+See Uninstall Docker Desktop for:
```
```suggestion
To uninstall Docker Desktop, refer to the following topics for OS-specific instructions:
```

usha-mandya commented on compose/install/uninstall.md:
```diff
@@ -0,0 +1,58 @@
+* [Linux](../../desktop/linux/install.md/#uninstall-docker-desktop){:target="_blank" rel="noopener" class="_"}
+
+
+### Uninstalling Compose CLI plugin
```
```suggestion
### Uninstalling the Compose CLI plugin
```

usha-mandya commented on compose/install/uninstall.md:
````diff
@@ -0,0 +1,58 @@
+```console
+$ sudo apt-get remove docker-compose-plugin
+```
+Or, if using a different distro, use the equivalent package manager instruction to remove `docker-compose-plugin`. 
````
Add a blank line above

usha-mandya commented on compose/install/uninstall.md:
```diff
@@ -0,0 +1,58 @@
+
+__Manually installed__
+
+If you used `curl` to install Compose CLI plugin, to uninstall it run:
```
```suggestion
If you previously used `curl` to install the Compose CLI plugin, to uninstall Docker Compose, run:
```

usha-mandya commented on compose/install/uninstall.md:
````diff
@@ -0,0 +1,58 @@
+```
+{% endraw %}
+
+to inspect the location of the Compose CLI plugin.
````
This looks like a dangling sentence. Can we combine it with the sentence above the code

usha-mandya commented on compose/install/uninstall.md:
```diff
@@ -0,0 +1,58 @@
+to inspect the location of the Compose CLI plugin.
+
+
+> Got a "Permission denied" error?
```
```suggestion
> Got a **Permission denied** error?
```

usha-mandya commented on compose/install/uninstall.md:
```diff
@@ -0,0 +1,58 @@
+
+> Got a "Permission denied" error?
+>
+> If you get a "Permission denied" error using either of the above
```
```suggestion
> If you get a **Permission denied** error using either of the above
```

usha-mandya commented on compose/install/index.md:
```diff
@@ -0,0 +1,38 @@
+toc_max: 2
+---
+
+In this page you can find a summary of the available options for installing Compose. 
```
```suggestion
This page contains a summary of various options available to install Docker Compose. 
```

usha-mandya commented on compose/install/uninstall.md:
```diff
@@ -0,0 +1,58 @@
+
+### Uninstalling Docker Desktop
+
+If you want to uninstall Compose and you have installed Docker Desktop, follow the corresponding link bellow to get instructions on how to remove Docker Desktop. 
```
```suggestion
To uninstall Compose installed through Docker Desktop, follow the corresponding link bellow to get instructions on how to uninstall Docker Desktop. 
```

usha-mandya commented:
@dockertopia I've suggested changes to the files. Looks like there are conflicting files as well. Could you also take a look at these?

dockertopia commented on compose/install/compose-plugin.md:
```diff
@@ -0,0 +1,194 @@
+
+## Installing Compose on Linux systems
+
+In this section you can find various routes for installing Compose in Linux.
```
Changing routes to methods because the next header includes "methods".

StefanScherer commented on compose/install/uninstall.md:
````diff
@@ -0,0 +1,58 @@
+
+{% raw %}	
+```console
+docker info --format '{{range .ClientInfo.Plugins}}{{if eq .Name "compose"}}{{.Path}}{{end}}{{end}}'
````
Adding a shell prompt probably improves the colored markdown.

```suggestion
$ docker info --format '{{range .ClientInfo.Plugins}}{{if eq .Name "compose"}}{{.Path}}{{end}}{{end}}'
```

StefanScherer commented on compose/install/compose-plugin.md:
```diff
@@ -0,0 +1,194 @@
+    This command downloads the latest release of Docker Compose (from the Compose releases repository) and installs Compose for _the active user_ under `$HOME` directory. 
+    
+    > To install:
+    >* Docker Compose for _all users_ on your system, replace `~/.docker/ cli-plugins` with `/usr/local/lib/docker/cli-plugins`.
```
Found a minor typo in the first directory, removing one space.

```suggestion
    >* Docker Compose for _all users_ on your system, replace `~/.docker/cli-plugins` with `/usr/local/lib/docker/cli-plugins`.
```

dockertopia replied:
Thanks, for your comments Stefan. I have already negotiated each of Usha's suggestions offline. I have been using the "Resolve conversation" button bellow each comment to mark changes I have already addressed in my local branch before committing. So all comments are still present on the page but just collapsed.
I will add your improvements as well and then address conflicts.

StefanScherer replied:
I also can't see this change. Please take another look.

StefanScherer reviewed (changes requested):
Found two tiny things.
I looked through all other suggestions which also make sense. Just a note to myself I had to click the "xx hidden conversations, Load more...` link to see more/all of them.

And yes the PR has merge conflicts, so it also needs some care.

dockertopia commented:
I believe I have addressed all comments and solved the merge conflicts, but please let me know if I missed anything. @usha, @StefanScherer could you PTAL. Feel completely free to add anything that you'd still like to discuss/see addressed. Thanks.

usha-mandya commented:
There are broken links in https://deploy-preview-14892--docsdocker.netlify.app/compose/install/#where-to-go-next. Details - https://github.com/docker/docker.github.io/runs/6814146716?check_suite_focus=true#step:4:460

usha-mandya reviewed (approved):
LGTM. Thank you @dockertopia

crazy-max commented on _data/toc.yaml:
```diff
@@ -1400,12 +1400,6 @@ manuals:
     title: Docker Scan
   - path: /engine/sbom/
     title: Docker SBOM (Experimental)
-
-- sectiontitle: Docker Build
-  section:
-    - path: /build/
-      title: Overview
-
```
Why as this been removed?

StefanScherer replied:
Good catch. Sorry, didn't recognize this change after the last commit. @dockertopia Could we bring this in again? Thanks.

dockertopia replied:
This will be merged to Published probably before. Or is Build section already on the master ready to be published? 
I understood that was going later on when everyone was good on a first version for Docker Build to be public.

dockertopia replied:
@crazy-max Does this make sense to you? My understanding is that this change in TOC will not affect a page/section that does exist in master yet?

crazy-max replied:
> Does this make sense to you?

I'm just surprised by this change in this PR because it's out of scope with its description. Would have been better for another PR so we can track this easily.

> I understood that was going later on when everyone was good on a first version for Docker Build to be public

Ok I misunderstood, I thought we wanted to iterate the changes and make them public as we go along.
